src/Server.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Protocol/ProtocolRecognizer.h"
 #include "CommandOutput.h"
 #include "FastRandom.h"
+#include "UUID.h"
```

Comment:
I think the include might be unnecessary - the object is only passed through by reference.